### PR TITLE
[CONTACTS] Create Contact model and migration - OT198-55

### PR DESCRIPTION
**Pull request by nicobes14**
[OT198-55](https://alkemy-labs.atlassian.net/browse/OT198-55)
## Description
HOW: As admin user
I WANT: Stores the data sent when completing a Contact Form
TO: Keep track of them

### Acceptance criteria:
- By completing the Contact Form, it will be stored in the database. Table name: contacts. It will contain the fields name, phone, email, message, deletedAt (used for soft delete)
#### Model fields:
id: INTEGER NOT NULL AUTO_INCREMENT
name: VARCHAR NOT NULL
phone: INTEGER NULLEABLE
email: VARCHAR NOT NULL
message: VARCHAR NOT NULL
softDelete
## Evidence
image

### CODE REVIEW

File: database/migrations/20220522235553-create-contact.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      phone: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
```
Comment (honeybadger2788) [inline database/migrations/20220522235553-create-contact.js:16]: En el modelo de Organizacion, el teléfono es NULLABLE. Acá entiendo que debería ser igual,no?

File: database/models/contact.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  Contact.init(
+    {
+      name: { type: DataTypes.STRING, allowNull: false },
+      phone: { type: DataTypes.INTEGER, allowNull: false },
```
Comment (honeybadger2788) [inline database/models/contact.js:17]: Idem comentario archivo migration

**Review by penalvapablo — APPROVED** [review]
Todo ok!

**Review by honeybadger2788 — APPROVED** [review]
Joya, Nico!

**Review by gabodmc — APPROVED** [review]
Buen pr 💪

**Pull request merged**